doc/model.rst

```diff
@@ -0,0 +1,683 @@
+and assign values to parameters used within the simulation via the `params`
+initialization argument. Parameters supplied in the `params` argument can either be
+written into supplied configuration files or be assigned within configuration files
+for use within the simulation at runtime. This functionality is supported by the
+``Model.attach_generator_files()`` helper function users have access to
+once creating a Model instance.
+
+SmartSim supports **two** strategies for deploying Models:
+
+- **Standard Model**: Operating on a separate compute node from a
+  SmartSim orchestrator, Standard Models facilitate communication with a clustered
+  Orchestrator through SmartRedis clients. This architecture is ideal for simulations
+  leveraging distributed computing capabilities, ensuring parallelized and efficient
+  execution across multiple nodes.
+
+- **Colocated Model**: Sharing compute resources with a colocated Orchestrator on the
+  same compute node, Colocated Models offer advantages in scenarios where minimizing communication
+  latency is critical, such as online inference or runtime processing.
+
+SmartSim manages ``Model`` instances through the :ref:`Experiment API<experiment_api>` by providing functions to
+launch, monitor, and stop simulations. Additionally, Models can be launched individually
+or as a group via an Ensemble. Once a Model instance has been initialized, users have access to
+the :ref:`Model API<model_api>` helper functions.
+
+======================
+Initialize Model Types
+======================
+--------
+Overview
+--------
+
+The :ref:`Experiment API<experiment_api>` is responsible for initializing all workflow components.
+A ``Model`` is created using the ``Experiment.create_model()`` helper function. Users can customize the
+the Model by specifying initializer arguments.
+
+The key initializer arguments are:
+
+-  `name` (Type: str): Specify the name of the model for unique identification.
+-  `run_settings` (Type: RunSettings): Describe execution settings for a Model.
+-  `params` (dict, optional): Provides a dictionary of parameters for Models.
+-  `path` (str, optional): Path to where the model should be executed at runtime.
+-  `enable_key_prefixing` (bool, optional): Prefix the model name to data sent to the database to prevent key collisions. Default is True.
+-  `batch_settings` (BatchSettings | None): Describes settings for batch workload treatment.
+
+To initialize a ``Model``, a `name` and :ref:`RunSettings<settings-info>` reference is required.
+To instruct a Model to encapsulate a simulation, users must provide the simulation
+executable to the Models run settings with instructions on how the application should be launched.
+
+Users may also launch Models as a batch job by specifying :ref:`BatchSettings<settings-info>` when initializing.
+When a Model with a ``BatchSettings`` reference is added to an Ensemble with a ``BatchSettings`` reference,
+the Models batch settings are strategically ignored.
```

Comment:
- Consider "BatchSettings attached to a model are ignored when it is executed as part of an ensemble." 
- consider adding as a [note](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes)